### PR TITLE
Add NodeJS v7 to package.json

### LINKED ISSUE #29: Add support for node 7
### Bug or feature request
- [ ] Bug
- [x] Feature request

### Description of feature (or steps to reproduce if bug)
Cannot install loopback-context with yarn as it complains about "incompatible" node engine: 
> `error loopback-context@3.1.0: The engine "node" is incompatible with this module. Expected version "^4.5 || ^5.10 || ^6.0".`

### Expected result
loopback-context to be compatible with NodeJS v7 or at least package.json to be updated

### Additional information (Node.js version, LoopBack version, etc)
Node v.7.7.1

### PR DESCRIPTION
**Pull request by xnf**
### Description
Added NodeJS v7 to package.json. 
This way yarn will not complain when node 7 is used.
Tests still pass and the package itself also works on Node 7.

#### Related issues

Fixes #29 Add support for Node 7

### Checklist
- [ ] New tests added or existing tests modified to cover all changes
- [X] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)

### CODE REVIEW

**Comment by slnode** [comment]
Can one of the admins verify this patch? To accept patch and trigger a build add comment ".*ok\W+to\W+test.*"

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Review by bajtos — CHANGES_REQUESTED** [review]
Hi @xnf, thank you for the pull request. Is there any particular reason why you are sticking to Node.js 7, when Node.js 8 has been already released? Would you mind including the version 8 in the list of supported versions too?

More importantly, please modify `.travis.yml` file to run CI tests on the newly added platforms, in order to verify that the module correctly works on them.

**Comment by xnf** [comment]
Omg, this collaboration flow is soo bad from my side. Sorry for that. 

Thing is that i have a server with v7 and just want to use loopback-context. Test for V8 failed. and as for me - i don't even need it.

**Comment by bajtos** [comment]
No worries! Let's add just Node.js 7 then. I have squashed your commits into a single one and improved the commit message a bit. Let's wait for CI results before landing.

**Review by bajtos — APPROVED** [review]

**Comment by bajtos** [comment]
Landed, thank you for the contribution 🎉

**Pull request merged**
